README.md

```diff
@@ -2,79 +2,79 @@
 
-1. Login to your GitHub account.
-2. Navigate to the Rocky Linux Documentation repository at: [https://github.com/rocky-linux/documentation.git](https://github.com/rocky-linux/documentation.git)
-3. You should be on the "Main" branch, so check the drop down label down in the middle section, just to be sure you are. Your document might not ultimately end up in the "Main" branch, but admins will move it around to where it logically fits later.  
-On the right-hand side of the page, click the "Fork" button, which will create your own copy of the documentation.
-4. In the middle of the page on the forked copy, just to the left of the Green "Code" drop down, is an "Add file" button. Click this and choose the "Upload files" option.
-5. This will give you a way to either drag and drop files here, or to browse to them on your computer. Go ahead and use the method you are most comfortable with.
-6. Once the file is uploaded, the next thing you need to do is create a Pull Request. This request lets the upstream administrators know that you've got a new file (or files) that you would like them to merge with the master.  
+1. log in to your GitHub account.
+2. Navigate to the Rocky Linux Documentation repository at [https://github.com/rocky-linux/documentation.git](https://github.com/rocky-linux/documentation.git)
+3. You should be on the "Main" branch, so check the drop-down label down in the middle section to be sure you are. Your document might not ultimately end up in the "Main" branch, but admins will move it around to where it logically fits later.  
+On the right-hand side of the page, click the "Fork" button, which will create your copy of the documentation.
+4. In the middle of the page on the forked copy, just to the left of the Green "Code" drop-down, is an "Add file" button. Click this and choose the "Upload files" option.
+5. This will give you a way to either drag and drop files here or browse to them on your computer. Go ahead and use the method which you prefer.
+6. Once the file is uploaded, the next thing you need to do is create a Pull Request. This request lets the upstream administrators know that you have a new file (or files) that you would like them to merge with the master.  
 Click on "Pull Request" in the upper-left of the screen.
 7. Write a brief message in the "Write" section letting the administrators know what you've done. (New document, revision, suggested change, etc.,) then submit your change.
 
 ### From the Git Command-Line
 
-If you prefer to run Git locally on your own machine, you can clone the Rocky Linux Documentation repository, make changes, and then do a commit changes afterwards. To make things super-easy, do steps 1-3 on *From Git Gui* above then:
+If you prefer to run Git locally on your machine, you can clone the Rocky Linux Documentation repository, make changes, and then commit changes afterward. To make things simple, execute steps 1-3 on *From Git Gui* above then:
```

Comment:
Change Gui to GUI I missed it.

Reply:
Can I make a suggestion? When editing another document that has been contributed to by several people, try not to change the "voice" of the document, even though it may not be something that you would write. Here, you've made some very good corrections and suggestions, but one stands out as changing the voice. For instance, there is nothing technically or grammatically wrong with "super-easy" except of course the hyphen. It should be "super easy". This leaves the voice of the original author(s) in tact, while changing the things that are in need of correcting. Just a suggestion.

Reply:
Super-easy is useless because it does not translate in most languages. The problem here exactly is the voice. Too much usage of passive voice instead of active voice. Documentation should have a neutral voice; passive voice is for people doing social work and therapy, not for instruction. By the way, what does super easy mean/? I actually use super easy when I am writing fiction but I use it thus as "super easy".

Reply:
In essence, there should be no use of passive voice or colloquial verbiage in the documentation. Voice should be neutral and grammar should be standard and simple. I can write contorted sentences that are grammatically correct by that often lowers the ability of the reader to quickly understand what we are saying..

Reply:
I have no horse in this race, I am just copyediting for ease of understanding and optimization for translation. Anything extraneous does not belong there. If something is easy to do, super easy does not increase the reader's understanding of it.


In closing, while voice is part of meaning and semantics, voice use other than neutral does not belong in technical writing or documentation. There are many good reasons for this especially once the documentation base grows in size. We either get a handle on this early or deal with it at a later time with greater difficulty. Plus it will help people enhance their understanding of Grammar. Something which I found in my university days to be sorely lacking. Grammatical usage is dependant on the writing environment. Academic papers, writing fiction, poetry, these all have different parameters and what works well for fiction probably will be sub-optimal for technical writing (including documentation).

Each of the changes has not been done casually, much thought has gone into how each change affects the holistic understanding of the entire document. This with on eye on what I think to be of great importance...Making the job of the translator as optimal as is possible.

Best wishes,
Gerry